pkg/applicationserver/applicationserver.go

```diff
@@ -1049,6 +1049,14 @@ func (as *ApplicationServer) handleJoinAccept(
 		return err
 	}
 
+	entity, err := as.endDeviceRegistry.Get(ctx, ids, []string{"attributes"})
+	if err != nil {
+		log.FromContext(ctx).WithError(err).
+			Warn("Failed to retrieve end device attributes on join-accept")
```

Comment:
Nit; line breaks, should be
```suggestion
		log.FromContext(ctx).WithError(err).Warn(
			"Failed to retrieve end device attributes on join-accept",
		)
```

Reply:
done